tool/release/Bump.kt

```diff
@@ -0,0 +1,48 @@
+package tool.release
+
+import com.google.api.client.http.ByteArrayContent
+import com.google.api.client.http.GenericUrl
+import com.google.api.client.http.HttpHeaders
+import com.google.api.client.http.javanet.NetHttpTransport
+import java.nio.file.Files
+import java.nio.file.Paths
+
+
+fun postJson(url: String?, authorization: String, accept: String, content: String) {
+    NetHttpTransport()
+    .createRequestFactory()
+    .buildPostRequest(GenericUrl(url), ByteArrayContent.fromString("application/json", content))
+    .setHeaders(HttpHeaders().setAuthorization(authorization).setAccept(accept))
+    .execute()
+}
+
+fun main() {
+    val workspaceDirectory = System.getenv("BUILD_WORKSPACE_DIRECTORY")
+            ?: throw RuntimeException("Not running from within Bazel workspace")
+    val githubToken = System.getenv("GITHUB_TOKEN")
+            ?: throw RuntimeException("GITHUB_TOKEN environment variable is not set")
+    val repoOwner = System.getenv("GRABL_OWNER")
+            ?: throw RuntimeException("GRABL_OWNER environment variable is not set")
+    val repoName = System.getenv("GRABL_REPO")
+            ?: throw RuntimeException("GRABL_REPO environment variable is not set")
+
+    val versionFile = Paths.get(workspaceDirectory, "VERSION")
+    val content = String(Files.readAllBytes(versionFile)).trim()
+    val versionComponents = content.split(".").map { it -> Integer.parseInt(it)}.toTypedArray()
+
+    if (versionComponents.size != 3) {
```

Comment:
our versioning is quite free now. can we update the validation to this regex: https://github.com/graknlabs/bazel-distribution/blob/master/maven/templates/deploy.py#L116

Reply:
- if the version looks like `2.0.0`, bump it to `2.0.1`
- if it looks like `2.0.0-alpha-1` bump it to to `2.0.0-alpha-2`

Reply:
@lolski addressed in https://github.com/graknlabs/dependencies/pull/237/commits/b1039df806934c93deb04cee8be6f435f0927244